maps/away/russianderelict/russianderelict.dm

```diff
@@ -252,3 +259,34 @@
 
 /obj/effect/floor_decal/urist/russian
 	icon_state = "derelict1"
+
+//contract
+
+/datum/contract/russianderelict
+	name = "Derelict Repower Contract"
+	desc = "Restore power to this station by setting up the singularity."
```

Comment:
This is the description that shows up in the contract database, so this should probably be expanded on a little. I had humoured the thought of adding in a USSSR faction datum to pay the money out from, but it's up to you whether you want it to be an NT contract or not. If you think that sounds neat, i'll add it in right now and you can set the contract faction as "russian". Otherwise you don't need to set the faction, because it autosets it as GLOB.using_map.trading_faction if one isn't set.

Reply:
It's a neat concept but I don't think this contract alone justifies a new faction that otherwise has very little lore presence compared to our other ones.